Assets/Scripts/MainGame/MinimapBaygate.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections;
```

Comment:
these location minimap scripts all have the exact same code just call a different location from the enum. I'd consider making a single class like minimapLocator, and then just have a single public variable exposed for the editor to set the area to go show.